test/src/ui/image_picker_ui_test.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:typed_data';
```

Comment:
dit bestand mag direct in de test folder, hoeft niet in test/src/ui

Reply:
Aangepast